docs/api_reference/requirements.txt

```diff
@@ -3,7 +3,7 @@
 -e libs/core
 -e libs/community
 pydantic<2
-autodoc_pydantic==1.8.0
+autodoc_pydantic==1.9.0
```

Comment:
whats the issue with 1.8.0?

Reply:
I get this below error with 1.8.0 which doesn't happen with 1.9.x

```bash
API reference files built.
cd docs/api_reference && poetry run make html
Running Sphinx v4.5.0

Extension error:
Could not import extension sphinxcontrib.autodoc_pydantic (exception: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.6/migration/#basesettings-has-moved-to-pydantic-settings for more details.

For further information visit https://errors.pydantic.dev/2.6/u/import-error)
make[1]: *** [html] Error 2
make: *** [api_docs_build] Error 2
```